### PR TITLE
fix: Added V4 RFQ order schema

### PR DESCRIPTION
**Pull request by PirosB3**
## Description

Adds V4 RFQ order schema and V4 signature

### CODE REVIEW

File: json-schemas/test/schema_test.ts [hunk 1]
```diff
@@ -150,6 +151,35 @@ describe('Schema', () => {
             validateAgainstSchema(testCases, blockRangeSchema, shouldFail);
         });
     });
+    describe('#V4 RFQ Schema', () => {
+        const sampleTransaction = `
```
Comment (alexkroeger) [inline json-schemas/test/schema_test.ts:155]: naming nit, not a transaction

**Review by alexkroeger — APPROVED** [review]

File: json-schemas/test/schema_test.ts (hunk 1 above)
Comment (PirosB3) [reply json-schemas/test/schema_test.ts:155]: ugh! nice

**Review by dorothy-zbornak — APPROVED** [review]
:+1:

**Comment by dorothy-zbornak** [comment]
Think you need a rebase to fix that publish job.

**Pull request merged**